modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -360,6 +360,13 @@
 			}
 			this._super( property, value );
 		},
+		updateCaptionsMenu: function () {
+			var _this = this;
```

Comment:
@ErezBouskila this is not required for embedded captions so why add it?

Reply:
In 'showEmbeddedCaptions' the trigger to creating and populating the menu is from the event `textTracksReceived`
With this change we are new creating an empty menu on playerReady when we have embedded captions.

Reply:
@ErezBouskila you need to just destroy the menu on changeMedia
Rebuilding it is handled separately for each mode and you need to keep it like that!